tests/test_webvtt_conversion.py

```diff
@@ -34,8 +36,10 @@ def test_positioning_is_kept(self):
         caption_set = WebVTTReader().read(
             SAMPLE_WEBVTT_FROM_DFXP_WITH_POSITIONING)
         results = WebVTTWriter().write(caption_set)
+	# TODO: Change this to SAMPLE_WEBVTT_FROM_DFXP_WITH_POSITIONING once
```

Comment:
Hi, if I don't change the WebVTT test case, I will have to change the DFXP test case test_dfxp_with_positioning_to_webvtt_conversion. Would that be alright?